client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -0,0 +1,277 @@
+import { TITAN_MAIL_MONTHLY_SLUG } from '@automattic/calypso-products';
+import { Gridicon } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import { withShoppingCart } from '@automattic/shopping-cart';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import page from 'page';
+import React, { FunctionComponent, useState } from 'react';
+import { connect } from 'react-redux';
+import poweredByTitanLogo from 'calypso/assets/images/email-providers/titan/powered-by-titan-stacked.svg';
+import PromoCardPrice from 'calypso/components/promo-section/promo-card/price';
+import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+import { titanMailMonthly } from 'calypso/lib/cart-values/cart-items';
+import {
+	getSelectedDomain,
+	canCurrentUserAddEmail,
+	getCurrentUserCannotAddEmailReason,
+} from 'calypso/lib/domains';
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
+import { getTitanProductName, isDomainEligibleForTitanFreeTrial } from 'calypso/lib/titan';
+import { TITAN_PROVIDER_NAME } from 'calypso/lib/titan/constants';
+import {
+	areAllMailboxesValid,
+	buildNewTitanMailbox,
+	transformMailboxForCart,
+	validateMailboxes as validateTitanMailboxes,
+} from 'calypso/lib/titan/new-mailbox';
+import withCartKey from 'calypso/my-sites/checkout/with-cart-key';
+import EmailProvidersStackedCard from 'calypso/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card';
+import {
+	TITAN_PASSWORD_RESET_FIELD,
+	TITAN_FULL_NAME_FIELD,
+} from 'calypso/my-sites/email/titan-new-mailbox';
+import TitanNewMailboxList from 'calypso/my-sites/email/titan-new-mailbox-list';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getProductBySlug, getProductsList } from 'calypso/state/products-list/selectors';
+import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { EmailProvidersStackedCardProps, ProviderCard } from './provider-card-props';
+import type { TranslateResult } from 'i18n-calypso';
+
+import './professional-email-card.scss';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+const logo = <Gridicon className="professional-email-card__logo" icon="my-sites" />;
+const badge = <img src={ poweredByTitanLogo } alt={ translate( 'Powered by Titan' ) } />;
+const getTitanFeatures = () => {
+	return [
+		translate( 'Inbox, calendars, and contacts' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Email, calendars, and contacts' )` *ES Score: 6*